src/dscim/preprocessing/input_damages.py

```diff
@@ -744,6 +745,10 @@ def prep_mortality_damages(
         scaling_deaths = "epa_row"
         scaling_costs = "epa_scaled"
         valuation = "vsl"
+    elif mortality_version == 9:
```

Comment:
Mortality version 9 is a new one to me. Need to add this to our documentation. Also, we need to add brief comments in here about what each version means (or at least what the string values mean).